build.sh

```diff
@@ -112,6 +114,49 @@ build_managed_wcf()
     echo Build Exit Code = $BUILDERRORLEVEL
 }
 
+build_native_corefx()
```

Comment:
No.  We need to expunge the native command line switches and script logic. I'm afraid this has to remain one of the diffs we need to ignore each time we sync.  Or find some creative way to indicate "this is CoreFx logic only" that makes it a NOP in our repo.